hedera-node/src/main/java/com/hedera/services/context/ServicesContext.java

```diff
@@ -1258,16 +1291,27 @@ public void updateFeature() {
 		}
 	}
 
-	public RecordStream recordStream() {
-		if (recordStream == null) {
-			recordStream = new RecordStream(
+	public RecordStreamManager recordStreamManager() {
+		return recordStreamManager;
+	}
+
+	/**
```

Comment:
Don't need to change for this PR, but if you want the ability to dynamically override record streaming (i.e. without a network restart), we would want to inject the `RecordStreamManager` with the `GlobalDynamicProperties` singleton.